packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -567,6 +587,8 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
   ///
   /// This value can also be obtained over the VM service protocol as
   /// `ext.flutter.didSendFirstFrameEvent`.
+  ///
+  /// See also [firstFrameRasterized].
```

Comment:
style for "see also" is:

```suggestion
  /// See also:
  ///
  ///  * [firstFrameRasterized], <reason why I should look there>.
```

Reply:
Done.